phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
```

Comment:
I don't really think it is necessary to be able to complete these steps in any order, as if i'm understanding correctly it is possible to go forward and backward with your modification.

Also, it would be probably fine if you just let the user jump back to the the part where the install starts. However, in that case as well we need to clear stuff out from the database etc. It is not the simplest issue, that's for sure.